src/app/personside/NyIdentModal.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import { setNyBrukerIPath } from '../routes/routing';
+import { useHistory } from 'react-router';
+import RawModal from 'nav-frontend-modal';
+import KnappBase from 'nav-frontend-knapper';
+import styled from 'styled-components/macro';
+import { Systemtittel } from 'nav-frontend-typografi';
+import { ReactComponent as ErrorIkon } from '../../svg/alvorlig-advarsel.svg';
+import { useState } from 'react';
+
+const Modal = styled(RawModal)`
+    text-align: center;
+    min-width: 20rem;
+`;
+const Knapp = styled(KnappBase)`
+    width: 100%;
+`;
+
+interface NyIdentTilgangProps {
+    aktivIdent: string;
+}
+
+function NyIdentModal(props: NyIdentTilgangProps) {
+    const history = useHistory();
+    const [open, setOpen] = useState(true);
+
+    const handleClick = () => {
+        setOpen(true);
+        setNyBrukerIPath(history, props.aktivIdent);
+    };
+
+    return (
+        <Modal
+            isOpen={open}
+            contentLabel="NyIdent"
+            closeButton={false}
+            shouldCloseOnOverlayClick={false}
+            onRequestClose={() => {
+                setOpen(false);
+            }}
+        >
+            <ErrorIkon width="2rem" className="blokk-xs" />
+            <Systemtittel tag="h1" className="blokk-xs" role="alert" aria-live="assertive">
+                Oppslag på inaktiv ident
+            </Systemtittel>
+
+            <div className="noncenter typo-normal blokk-m">
+                {`Personidenten du slo opp er ikke lenger aktive. Ny ident for bruker er ${props.aktivIdent}.`}
+            </div>
```

Comment:
Trenger ikke å bruke string templates her
```suggestion
            <div className="noncenter typo-normal blokk-m">
                Personidenten du slo opp er ikke lenger aktive. Ny ident for bruker er {props.aktivIdent}.
            </div>
```